core/src/main/java/io/grpc/Server.java

```diff
@@ -38,4 +41,57 @@
  * application code or interceptors.
  */
 @ThreadSafe
-public abstract class Server {}
+public abstract class Server {
```

Comment:
Well, for starters, we do have a `Server`... `ManagedChannel` was replacing `ChannelImpl`. The fate of the `Server` class has always been foggy. It's not clear to me if it will gain methods in the future. So it seems reasonable to leave room for a separate `Server` abstract class (even if it doesn't exist today).